x-pack/test/performance/journeys/base.config.ts

```diff
@@ -36,7 +36,18 @@ export default async function ({ readConfigFile, log }: FtrConfigProviderContext
     },
     kbnTestServer: {
       ...functionalConfig.get('kbnTestServer'),
-      serverArgs: [...functionalConfig.get('kbnTestServer.serverArgs')],
+      serverArgs: [
+        ...functionalConfig.get('kbnTestServer.serverArgs'),
+        `--telemetry.labels.branch=${process.env.BUILDKITE_BRANCH}`,
+        `--telemetry.labels.ciBuildId=${process.env.BUILDKITE_BUILD_ID}`,
+        `--telemetry.labels.ciBuildJobId=${process.env.BUILDKITE_JOB_ID}`,
+        `--telemetry.labels.ciBuildNumber=${process.env.BUILDKITE_BUILD_NUMBER}`,
+        `--telemetry.labels.git_rev=${process.env.BUILDKITE_COMMIT}`,
+        `--telemetry.labels.isPr=${process.env.CI}`,
```

Comment:
Why does running in CI indicate that this is a PR?

Reply:
Ohh this part was copied from APM labels and there was an issue with this label as you mentioned which I found last week and [fixed](https://github.com/elastic/kibana/pull/136869)but forgot to update this PR. Thanks 👍🏽